app/models/document.rb

```diff
@@ -1,7 +1,7 @@
 class Document < ApplicationRecord
   belongs_to :application
 
-  has_attached_file :attachment, PAPERCLIP_STORAGE_OPTIONS 
-  validates_attachment :attachment, content_type: { content_type: ['application/pdf', 'application/msword', 'application/xml', /\Aapplication\/vnd.openxmlformats-.*/, 'image/jpg', 'image/jpeg', 'image/png', 'image/gif'] }
-  validates :attachment_file_name, presence: true
+  # has_attached_file :attachment, PAPERCLIP_STORAGE_OPTIONS
+  # validates_attachment :attachment, content_type: { content_type: ['application/pdf', 'application/msword', 'application/xml', /\Aapplication\/vnd.openxmlformats-.*/, 'image/jpg', 'image/jpeg', 'image/png', 'image/gif'] }
+  # validates :attachment_file_name, presence: true
```

Comment:
To clarify, by "that test" I mean the first one to test the has_many documents relationship in application_spec.rb (specifically transcripts in this case).  All document validations after the first one work for some reason.